### DISCUSSION TITLE
Add new column on Users' Section

### DISCUSSION THREAD

**Discussion by oculos** [keycloak/keycloak#29442] [Q&A]
Hi,

I added an attribute to the User Profile, and I'd like it to be displayed o the Users' list.
I copied the "UsersSecti

on-8tD7bbqw.js" from the admin-ui (keycloak.v2) into my theme, and added the column. The problem is that while the title of the column was added, it was populated with the "username" attribute, not with the attribute I created (called "origin"). 

I also checked UsersSection-8tD7bbqw.js.map, but didn't get where exactly I should add this new attribute. 
Could someone give me a hint? 

<img width="892" alt="Screenshot 2024-05-10 at 10 20 34" src="https://github.com/keycloak/keycloak/assets/6791923/776304a8-5fe0-4d55-8700-93554779efcb">

**Comment by fradrick** [comment]
Did you get a breakthrough?

**Reply by oculos** [reply]
Not really. It seems that the admin client fetches data on this section from an api that doesn’t return all the attributes. So I’m stuck.
